botocore/utils.py

```diff
@@ -524,8 +525,9 @@ class ArgumentGenerator(object):
         print("Sample input for dynamodb.CreateTable: %s" % sample_input)
 
     """
-    def __init__(self):
-        pass
+    def __init__(self, use_member_names=False):
+        self._use_member_names = use_member_names
```

Comment:
Is there a particular reason why you made this ``use_member_names`` other than to get a truthy value? If so I wonder if it makes sense to expose a mapping of shape to default value or ``use_truthy_defaults`` optional parameter that defaults to ``True``?